repos/fdbt-site/src/components/TapPricingRow.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ReactElement } from 'react';
+import { ErrorInfo, MultiTap } from '../interfaces';
+import FormElementWrapper, { FormGroupWrapper } from './FormElementWrapper';
+
+interface TapPricingRowProps {
+    numberOfTapsToDisplay: number;
+    errors: ErrorInfo[];
+    userInput: MultiTap;
+}
+
+export const renderTable = (index: number, errors: ErrorInfo[], userInput: MultiTap = {}): ReactElement => (
+    <fieldset key={index} className="govuk-fieldset">
+        <legend className="govuk-fieldset__legend govuk-visually-hidden">{`Enter details for tap ${index + 1}`}</legend>
+        <div className="govuk-grid-row govuk-!-width-one-third govuk-!-margin-left-4">
+            <div className="govuk-grid-row">
+                <div className="govuk-grid-column-one-half">&nbsp;</div>
```

Comment:
What is this div doing?

Reply:
for creating the columns ... like a table of rows and columns

Reply:
I don' think this is correct, https://design-system.service.gov.uk/styles/layout/ this is how to use grid.